### DISCUSSION TITLE
githubConfigUrl in actions-runner-controller-charts/gha-runner-scale-set Helm Chart

### DISCUSSION THREAD

**Discussion by rwlove** [actions/actions-runner-controller#3250] [Questions]
I'm not sure what should be in this repository and I've struggled to find documentation stating what should be there. Is there some documentation or better yet an example I can build from?

https://raw.githubusercontent.com/actions/actions-runner-controller/master/charts/gha-runner-scale-set/values.yaml

```
## githubConfigUrl is the GitHub url for where you want to configure runners
## ex: https://github.com/myorg/myrepo or https://github.com/myorg
githubConfigUrl: ""
```

**Comment by shaul75** [comment]
That is simply the repository in which the runners will run in

**Comment by AdonisShabaniRB** [comment]
You are supposed to input the URL of the Repo or Organization or Enterprise. If you want to share only with a repo than use your repo URL if you want to share the runner with the Organization than use the Organization URL and so on.

**Reply by rwlove** [reply]
I'm using self hosted runners for my Home Ops repo, so I suppose this should point to my Home Ops repo?

**Comment by harsh-p-soni** [comment]
@rwlove Please check below document and code snippet to answer your question. If you want register your runner against `Org` then till `Org` you should use the URL and if you want to register your runner against specific `repository` then you should use URL till your `repository`

https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller#configuring-a-runner-scale-set

INSTALLATION_NAME="arc-runner-set"
NAMESPACE="arc-runners"
GITHUB_CONFIG_URL="https://github.com/<your_enterprise/org/repo>"
GITHUB_PAT="<PAT>"
helm install "${INSTALLATION_NAME}" \
    --namespace "${NAMESPACE}" \
    --create-namespace \
    --set githubConfigUrl="${GITHUB_CONFIG_URL}" \
    --set githubConfigSecret.github_token="${GITHUB_PAT}" \
    oci://ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set

**Reply by singhujjwal** [reply]
can we point this to set of wildcard repo where we want the runner to be available e.g. `https://github.com/<my-org>/myteam.*`
As my org has many teams and I dont want other teams to be able to use my runners

**Reply by harsh-p-soni** [reply]
AFAIK wildcard will not work to register runner at repository level but runner can be added at org level and use of it can be controlled by custom runner labels though if other teams use the same label then workflow will be able to access the runner.

https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners#adding-a-self-hosted-runner-to-a-repository

https://github.blog/2020-04-22-github-actions-community-momentum-enterprise-capabilities-and-developer-improvements/#share-self-hosted-runners-across-an-organization
